build.gradle

```diff
@@ -562,6 +578,48 @@ project('test:system') {
 
         onlyIf { dockerRegistryUrl && repoUrl }
     }
+
+    task execShellScript(type: Exec) {
```

Comment:
I tried moving it into its own directory ` test/system` , but then we need to import all the dependencies, configurations separately  and we need to place this only into a file with  name  `build.gradle`. Also, moving into gradle sub directory doesnt work , also the current files there are configs, checkstyle and all other factors needed to build all projects instead of single project .